osu.Game/Overlays/Music/MusicKeyBindingHandler.cs

```diff
@@ -30,17 +30,20 @@ public class MusicKeyBindingHandler : Component, IKeyBindingHandler<GlobalAction
         [Resolved(canBeNull: true)]
         private OnScreenDisplay onScreenDisplay { get; set; }
 
+        [Resolved]
+        private OsuGame game { get; set; }
+
         public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
             if (e.Repeat)
                 return false;
 
-            if (beatmap.Disabled)
-                return false;
-
             switch (e.Action)
             {
                 case GlobalAction.MusicPlay:
+                    if (game.LocalUserPlaying.Value)
+                        return false;
```

Comment:
While this works for the cases that this PR aims to permit, notably `LocalUserPlaying` becomes false during pauses:

https://github.com/ppy/osu/blob/80fc13fc1f2952a1b376f8c73429f5864c95566f/osu.Game/Screens/Play/Player.cs#L448-L453

leading to stuff like this potentially happening:

https://user-images.githubusercontent.com/20418176/155597361-f4a61f72-a6ba-4675-85de-b2019503081d.mp4

It's not game-breaking but it does seem a bit off to me that the OSD popup shows up...? Not sure.

Reply:
Yeah, I'm not sure. But it doesn't break anything and is in line with being able to pause/play from the now playing overlay in the same state (and having the button update) so I don't see it as a huge issue for now.